.github/workflows/build-artifacts.yml

```
@@ -2766,14 +2766,128 @@ jobs:
+          python -m ipykernel install --user --name dih-project-kernel --display-name "DIH Project"
+
+      - name: Cache TinyTeX
+        uses: actions/cache@v4
+        with:
+          path: ~/.TinyTeX
+          key: tinytex-${{ runner.os }}-${{ hashFiles('.github/workflows/build-artifacts.yml') }}
+          restore-keys: |
+            tinytex-${{ runner.os }}-
+
+      - name: Install LaTeX
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2211:warning:4:1: This is a glob used as a command name. Was it supposed to be in ${..}, array, or is it missing quoting?

(shellcheck)